pkg/common/prometheus/metrics.go

```diff
@@ -127,4 +127,16 @@ var (
 		// Possible optype - "create-volume", "delete-volume", "attach-volume", "detach-volume", "expand-volume", etc
 		// Possible status - "pass", "fail"
 		[]string{"optype", "status"})
+
+	// AccessibleVolumesGauge is a gauge metric to observe the number of accessible volumes.
+	AccessibleVolumesGauge = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
Is it possible to expose single gauge metric with `accessible` & `inaccessible` labels? IMO that would also be better when creating a dashboard as data can be exposed over a single dashboard in Grafana/Wavefront.
If that's not an option in Prometheus, then this looks fine.

Reply:
@gohilankit  Dont think its possible, as far as i can tell. I've tried playing around with a single Gauge metric but no luck.

Reply:
I think it should be doable with `GaugeVec`. Can you explore https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#GaugeVec, and see if it's possible?

Reply:
done!